packages/rich-text/src/store/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { registerStore } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import reducer from './reducer';
+import * as selectors from './selectors';
+import * as actions from './actions';
+
+registerStore( 'core/formats', { reducer, selectors, actions } );
```

Comment:
It should be `core/rich-text` namespace.

Reply:
Ok